tests/metrics/network/README.md

```diff
@@ -0,0 +1,75 @@
+# Networking tests for the Intel® Clear Containers runtime
+
+## Introduction
+
+Currently for the Clear Containers project, we have a series of network performance tests 
+that can be used as a basic reference in order to measure network essentials like:
+
+- Bandwidth
+- Latency
+- Jitter
+- Throughput
+- CPU % and memory consumption
+
+## Performance tools
+
+- Iperf/Iperf3 is a simple tool to let you measure the bandwidth and the quality of a network 
+link. Iperf is multi-threaded and iperf3 is single threaded.
+
+- Nuttcp is a network performance tool that helps to determine the raw UDP network layer 
+throughput.
+
+- Smem is a memory reporting tool thats gives a user diverse reports on memory usage.
+
+## Networking tests
+
+- network-metrics
+
+Measures bidirectional network bandwidth using iperf (multi-threaded). Bidirectional tests are 
+used when we want to test both servers for the maximum amount of throughput.
+
+- network-latency
+
+Measures latency using ping. The ping utility can be used as a tool to measure how long 
+it takes one packet to get from one point to another.
+
+- network-metrics-iperf3
+
+Measures bandwidth, jitter and bidirectional bandwidth using iperf3 (single threaded). The 
+bandwidth test will help to show the rate of speed with which the data is transferred. On the other hand, 
+the jitter test will measure the variation in the delay of received packets.
+
+- network-metrics-nuttcp
+
+Measures UDP bandwidth using nuttcp. This will help to show the rate of speed which the data
+is transferred for the UDP protocol.
+
+- network-nginx-ab-benchmark
+
+Measures network performance using an nginx container and running the Apache benchmarking
+tool in the host to calculate the requests per second.
+
+- network-metrics-cpu-consumption
+
+Measures CPU % consumption while running maximum network bandwidth with iperf.
+
+- network-metrics-memory-pss
+
+Measures Proportional Set Size (PSS) memory using smem while running maximum network bandwidth
```

Comment:
`network-metrics-memory-pss` measures the Proportional Set Size (PSS) memory. It uses smem while running the maximum network bandwidth with iperf. PSS is a meaningful representation of the memory applications use.